starport/pkg/cosmosanalysis/msgs/msgs_test.go

```diff
@@ -0,0 +1,11 @@
+package msgs
+
+import (
+	"fmt"
+)
+
+func ExampleDiscover() {
```

Comment:
Wouldn't be possible to have unit tests where we have some templates that you save in tmpDir and then run the function under this dir?

Reply:
Yes, let's add this seperately.